pages/docs/input-data/crac/json/contingencies.md

```diff
@@ -44,15 +44,12 @@ crac.newContingency()
 ~~~
 {% endcapture %}
 {% capture t3_objects %}
-🔴⭐ **identifier**  
-⚪ **name**  
-⚪ **network elements**: list of 0 to N network elements  
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 🔴 **network element id**: must be the id of a PowSyBl network identifiable  
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; ⚪ **network element names**: names are optional, they can be used to make the CRAC
-more understandable from a business viewpoint, but applications relying on the CRAC do not necessarily need them.  
+The object used is the PowSyBl Contingency, please refer to the [dedicated documentation](https://www.powsybl.org/pages/documentation/index.html#grid-model) on PowSyBl website.
```

Comment:
since openrao defines the contingency adder API, better keep this documentation & add the required Type